pkg/loki/modules.go

```diff
@@ -222,6 +223,26 @@ func (t *Loki) initQuerier() (services.Service, error) {
 	t.Cfg.Worker.MaxConcurrentRequests = t.Cfg.Querier.MaxConcurrent
 
 	var err error
+	if t.Cfg.Querier.PostFilterChunk {
+		enc, err := chunkenc.ParseEncoding(t.Cfg.Ingester.ChunkEncoding)
+		if err != nil {
+			return nil, err
+		}
+		limiter := ingester.NewLimiter(t.overrides, nil, nil, t.Cfg.Ingester.LifecyclerConfig.RingConfig.ReplicationFactor)
+
+		postFetcherChunkFilterer := loki_storage.NewPostFetcherChunkFilterer(enc, t.Cfg.Ingester.BlockSize, t.Cfg.Ingester.TargetChunkSize, func(tenantId string) chunkenc.HeadBlockFmt {
+			var blockFmt chunkenc.HeadBlockFmt
```

Comment:
I think you can always use `chunkenc.OrderedHeadBlockFmt` because we always have ordered data in flushed chunks.

Reply:
done，👍 Thanks for your review and tips!